pipeline_tools/create_envelope.py

```diff
@@ -96,6 +101,35 @@ def create_submission_envelope(envelope_url, auth_headers, http_requests):
     return envelope_js
 
 
+def get_analysis_results(analyses_url, auth_headers, analysis_json_contents, http_requests):
+    """Checks the submission envelope for an analysis process with a protocol_id
+    that matches the analysis workflow id.
+
+    Args:
+        analyses_url (str): the url for creating the analysis record
+        auth_headers (dict): headers to use for auth
+        analysis_json_contents (dict): metadata describing the analysis
```

Comment:
Since you only use the workflow id, it would be clearer to pass only the workflow id in here and not the whole analysis json.